mantidproject/mantid
# Remove GPUAlgorithms

Removes the `GPUAlgorithms` library which has gone unused for a number of years and any CMake configuration relating to OpenCL.

**Report to:** nobody.

**To test:**

- See that build passes.

*There is no associated issue.*

*This does not require release notes* because it is unused code not exposed to users.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
